dashboard/app/views/levels/editors/fields/_widget_view.html.haml

```diff
@@ -0,0 +1,4 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#widget_view"}}
+  Widget View
+#widget_view.in.collapse
+  = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :widget_view, description: "Enable Widget View"}
```

Comment:
nit: might be helpful to have some helper text as to what this does (specifically the fact that it hides the editor workspace)?

Reply:
Oh yeah - will update!